langchain/src/prompts/index.ts

```diff
@@ -28,3 +28,4 @@ export {
   SerializedMessagePromptTemplate,
   SerializedChatPromptTemplate,
 } from "./serde.js";
+export { interpolateFString } from "./template.js";
```

Comment:
This is an implementation detail, which we do not want exported

Reply:
I often need to create my agents out of tree, which means implementing createPrompt which means people will want to be able to export this or something like it

Reply:
Yea, for that you should use `parseTemplate` or `renderTemplate` from the same file. I'd be happy to see those exported to everyone.